f8a_report/helpers/report_helper.py

```diff
@@ -64,32 +64,101 @@ def __init__(self):
 
         self.emr_api = os.getenv('EMR_API', 'http://f8a-emr-deployment:6006')
 
+    def cleanup_celery_task_metadata(self):
+        """Cleanup celery task metadata tables on a periodic basis."""
+        # Number of days to retain the celery task_meta data
+        num_days_metadata = os.environ.get('KEEP_DB_META_NUM_DAYS', '30')
+        # query to delete the celery task_meta data
+        query = sql.SQL('DELETE FROM celery_taskmeta '
+                        'WHERE DATE_DONE <= NOW() - interval \'%s day\';')
+        logger.info('Starting to clean up Celery Meta tables')
+        # Execute the query
+        self.cursor.execute(query.as_string(self.conn) % (num_days_metadata))
+        # Log the message returned from db cursor
+        logger.info('%r' % self.cursor.statusmessage)
+        logger.info('Cleanup of Celery Meta tables complete')
+
+    def cleanup_celery_worker_results(self):
+        """Cleanup worker results data tables on a periodic basis."""
+        # Number of days to retain the celery woker_result data
+        num_days_workerdata = os.environ.get('KEEP_WORKER_RESULT_NUM_DAYS', '30')
+        # query to delete the worker_result data
+        query = sql.SQL('DELETE FROM worker_results '
+                        'WHERE ended_at <= NOW() - interval \'%s day\';')
+        logger.info('Starting to clean up Worker Result data tables')
+        # Execute the query
+        self.cursor.execute(query.as_string(self.conn) % (num_days_workerdata))
+        # Log the message returned from db cursor
+        logger.info('%r' % self.cursor.statusmessage)
+        logger.info('Cleanup of Worker Result data tables complete')
+
+    def cleanup_package_analyses_data(self):
+        """Cleanup package analyses data tables on a periodic basis."""
+        # Number of days to retain the package_analyses data
+        num_days_package_analyses = os.environ.get('KEEP_PACKAGE_ANALYSES_NUM_DAYS', '30')
+        # query to delete the worker_result data
+        query = sql.SQL('DELETE FROM package_analyses '
+                        'WHERE ended_at <= NOW() - interval \'%s day\';')
+        logger.info('Starting to clean up Package Analyses data tables')
+        # Execute the query
+        self.cursor.execute(query.as_string(self.conn) % (num_days_package_analyses))
+        # Log the message returned from db cursor
+        logger.info('%r' % self.cursor.statusmessage)
+        logger.info('Cleanup of Package Analyses data tables complete')
```

Comment:
Can you make sure that the transaction gets committed after every SQL that is executed.